Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/inventory/StableRecyclerAdapter.kt

```diff
@@ -118,24 +116,38 @@ class StableRecyclerAdapter : androidx.recyclerview.widget.RecyclerView.Adapter<
             }
             ownedTextView.visibility = View.VISIBLE
             this.imageView.alpha = 1.0f
+            this.titleView.alpha = 1.0f
+            this.ownedTextView.alpha = 1.0f
+
             val imageName = if (itemType == "pets") {
-                "Pet-" + item.key
+                "Pet_Egg_" + item.animal
             } else {
                 "Mount_Icon_" + item.key
             }
-            this.ownedTextView.text = animal?.numberOwned?.toString()
-            ownedTextView.visibility = if (animal?.numberOwned == 0 || animal?.type == "special") View.GONE else View.VISIBLE
+
+            var owned = animal?.numberOwned?.toString()
+            var totalNum = animal?.totalNumber?.toString()
+            this.ownedTextView.text = context?.getString(R.string.pet_ownership_fraction, owned, totalNum)

```

Comment:
Why not Just have the owned and totalNum as integers here? The format string would need to use in that case `%d`.